recipes/CRISPRStudio/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version ="1" %}
+
+package:
+ name: crispr_studio
+ version: {{ version }}
+
+source:
+ - url: https://github.com/moineaulab/CRISPRStudio/archive/v1.tar.gz
+   sha256: 23cb08bdb851c473ba6275e40e9fdd9aedcb6590d1f53cc04cae776fb39e2fa8 
+ - url: http://faculty.virginia.edu/wrpearson/fasta/CURRENT/fasta-36.3.8g-linux64.tar.gz
+   sha256: 8aafc7cfff7286282439bafba327aa432ae4bba1917f2c96b0dc3e4433af4699
+   folder: fasta-36.3.8g
+ 
+build:
+ number: 0
+
+requirements:
+ build:
+  - {{ compiler('c') }} 
+ host:
+  - pip
```

Comment:
You're not using `pip`, you can omit this. Actually, you can omit the entire `host:` section.